packages/rn-tester/RNTesterPods.xcodeproj/project.pbxproj

```diff
@@ -893,6 +898,7 @@
 				CLANG_WARN__DUPLICATE_METHOD_MATCH = YES;
 				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Developer";
 				COPY_PHASE_STRIP = NO;
+				CXX = "";
```

Comment:
This change should be reverted.

Reply:
[8c7547e](https://github.com/facebook/react-native/pull/45114/commits/8c7547ec8823356c1578e79384ec159b1e10c817)